contracts/eosio.system/src/voting.cpp

```diff
@@ -120,6 +125,16 @@ namespace eosiosystem {
             },
             it->location
          );
+         current_it = it;
+      }
+
+      if (num_standby_slots > 0) {
+         for( auto it = ++current_it; it != idx.cend() && standby_producers.size() < num_standby_slots && 0 < it->total_votes && it->active(); ++it ) {
```

Comment:
for( auto it = ++current_it; it != idx.cend() && standby_producers.size() < num_standby_slots && 0 < it->total_votes && it->active(); ++it ) {